### ISSUE TITLE
add Claude's fragm-examples

### ISSUE THREAD

**Issue by claudeha** [3Dickulus/Fragmentarium_Examples_Folder#1]
I put some of mine here:
https://github.com/claudeha/fragm-examples

More to come...

**Comment by 3Dickulus** [comment]
I'll do that asap

**Comment by 3Dickulus** [comment]
ok, nicely done! I wish everyone would do that, the readme.md is included in the super-repo and thus the distribution packages as well. Before I go destroying what's there already I would like to verify  this assumption : All of the files in the current Examples/Claude folder are in your frag repo too so it's ok to just remove the current and replace with your frag repo.

```
drwxr-xr-x  2    198 Jun  7 16:13 ./
drwxr-xr-x 21   4096 Jun  7 18:47 ../
-rw-r--r--  1   1354 Jun  7 16:13 inflection-1.frag
-rw-r--r--  1   1283 Jun  7 16:13 inflection-2.frag
-rw-r--r--  1   485 Jun  7 16:13 MandelbarDE.frag
-rw-r--r--  1   25106 Jun  7 16:13 Math.frag
-rw-r--r--  1   4055 Jun  7 16:13 muet.frag
-rw-r--r--  1   1820 Jun  7 16:13 Triskelion.frag
-rw-r--r--  1   8210 Jun  7 16:13 Ultimate-AntiBuddhagram.frag
-rw-r--r--  1   1806 Jun  7 16:13 variations-1.frag

```

**Comment by claudeha** [comment]
Yes, I copied them in 30a050e7eecb82f06682d045316323764b10efcb

**Comment by 3Dickulus** [comment]
great...

**Comment by 3Dickulus** [comment]
done ;)

**Comment by claudeha** [comment]
I'm deleting my GH repos slowly, so please update the URL to https://code.mathr.co.uk/fragm-examples.git

**Comment by claudeha** [comment]
massive apologies for inconvenience - my site is attacked by robots, so I made a password a requirement - I guess it's best to revert the fragm-examples back to github...

**Comment by 3Dickulus** [comment]
no apologies required ;) 

I have a good list of blocking conditions for .htaccess (nginx) that covers most of the bad actors, it reduces the bot traffic and hack attempts to almost nil. It basically consists of data centres, hosting networks, VPNs and proxies. Happy to share that with you. I also redirect any requests from these bad actors back to the source IP so they are requesting the file from themselves, this tends to stop them from bothering my site(s) after a while.

put simply, deny everything, allow only what exists.

**Comment by claudeha** [comment]
no pw required any more (no web browse, only git tool allowed), so please apply this patch again:

```diff
From ec581399120a98660acaff382670e6ff1f6d9717 Mon Sep 17 00:00:00 2001
From: Claude Heiland-Allen <claude@mathr.co.uk>
Date: Wed, 26 Nov 2025 14:53:49 +0100
Subject: [PATCH] claude's fragm-examples moved from github to self-hosted

---
 .gitmodules | 2 +-
 Claude      | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/.gitmodules b/.gitmodules
index b73f38b..b4f853e 100755
--- a/.gitmodules
+++ b/.gitmodules
@@ -27,4 +27,4 @@
 	url = https://github.com/loicvdb/VDB-frags.git
 [submodule "Claude"]
 	path = Claude
-	url = https://github.com/claudeha/fragm-examples.git
+	url = https://code.mathr.co.uk/fragm-examples.git
diff --git a/Claude b/Claude
index a1964e9..79695a8 160000
--- a/Claude
+++ b/Claude
@@ -1 +1 @@
-Subproject commit a1964e983837f38e9d77b15f0950d75348ee9d06
+Subproject commit 79695a87f9ba719c2d3e1a08aca087dce0de6320
-- 
2.47.3

```
